### DISCUSSION TITLE
Interactive issue tree on /claims page

### DISCUSSION THREAD

**Discussion by olarubaj** [rufuspollock/awesome-crypto-critique#145] [Ideas]
> If I understood it correctly from my reading and conversations, the ideal issue tree would be interactive and support proposing new leafs for the tree and thus encourage more people to participate in exploring/researching web3.

### Exploring web3 claims

A user can explore claims that have been proposed and addressed so far on the tree. He can also find out which articles/podcasts/videos from Life Itself address a given claim and visit a given source upon clicking.

### Proposing web3 claims

A user can propose new claims and vote on claims proposed (not addressed yet) by others.

## Possible solution

- Tree body - root and leaves - in markdown file.
- The file will be parsed to JSON, which will then be used to generate an interactive D3 tree.
- The tree would enable zoom in/out on scrolling and/or clicking on a leaf, so that e.g. only a given branch is shown in detail
- Either each bottom level tree will show up a box with relevant Life Itself resources (clickable links, so that the user can go directly to the article/podcast and learn more) or bottom level leafs themselves will point to relevant resources (clickable links). 

## Ideal solution (?)

- a user can propose a new claim (or a change to it) directly via the tree -> adding a new leaf will automatically create a GitHub issue ?
- users can see claims already addressed by Life Itself but also not yet addressed and proposed by other users
- users can vote on claims proposed by others

> It would be cool if Life Itself would review claims proposed by users and address them in the form of articles/videos/podcasts.
>
> Maybe the proposed claims could be voted on by other users so that we know which of them are most interesting to the audience.

**Comment by theo-cox** [comment]
Great idea! For time I'd prioritise the interactive tree as a first pass, with editability/proposals in a v2 model
